keanu-project/src/main/java/io/improbable/keanu/model/regression/LinearRegressionGraph.java

```diff
@@ -67,7 +75,7 @@ public VertexId getInterceptVertexId() {
         return interceptVertex.getId();
     }
 
-    public VertexId getWeightsVertexId() {
+    public VertexId getWeightVertexId() {
```

Comment:
Since we're now exposing `getInterceptVertex` and `getWeightVertex`, can you remove the other methods from this class and `RegressionModel`, as they will only cause confusion:
1. `DoubleTensor getWeights()`
1. `double getIntercept()`
1. `VertexId getInterceptVertexId()`
1. `VertexId getWeightVertexId()`